USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -118,14 +189,65 @@ private string RenderMarker(Marker input)
                     output.AppendLine(mTMarker.Title);
                     output.AppendLine("</div>");
                     break;
+                case FMarker fMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker);
+                    }
+                    break;
+                case FTMarker fTMarker:
+                    string footnote="";
+                    foreach(Marker marker in input.Contents)
+                    {
+                        switch (marker)
+                        {
+                            case TextBlock textBlock:
+                                footnote += textBlock.Text+" ";
+                                break;
+                            case FQAMarker fQAMarker:

```

Comment:
FQA is actually an alternate translation so for consistency you should probably have footnote-alternate-translation as the class. But this works also.